frontend/src/features/rubricBuilder/RubricBuilder.tsx

```diff
@@ -367,12 +368,7 @@ export default function RubricBuilder(): ReactElement {
                 Import CSV
               </button>
 
-              <button
-                className="transition-all ease-in-out duration-300 bg-blue-600 text-white font-bold rounded-lg py-2 px-4 hover:bg-blue-700 hover:scale-105 focus:outline-none focus:ring-2 focus:ring-blue-500"
-                onClick={(event) => event.preventDefault()} // Placeholder
-              >
-                Export to CSV
-              </button>
+              <CSVExport rubric={rubric} />
```

Comment:
I love how you wrapped up all the logic inside the button! Can you rework the import code to do the same? That would clean up RubricBuilder a ton.